pr2_guide/Spot spot
---
std_msgs/Bool ret
